python/lsst/pipe/tasks/configurableActions/_configurableActionStructField.py

```diff
@@ -23,14 +23,16 @@
 __all__ = ("ConfigurableActionStructField", "ConfigurableActionStruct")
 
 from types import SimpleNamespace
-from typing import Iterable, Mapping, Optional, TypeVar, Union, Type, Tuple, List, Any, Dict
+from typing import (
+    Iterable, Mapping, Optional, TypeVar, Union, Type, Tuple, List, Any, Dict, Iterator, Generic)
```

Comment:
Maybe I'm just too used to `black` style, but I don't think this kind of indentation and close-parens are consistent with our style guide.

Reply:
You are correct and I meant to fix this. I had turned off black while working on this for fear of hitting all of pipe_tasks, and it shows how much I was used to it doing things already.